components/ui/article-detail.tsx

```
@@ -0,0 +1,18 @@
+import * as React from 'react'
+import type { Article } from '@/lib/types'
+
+interface Props {
+  article: Article
+}
+
+export default function ArticleDetail({ article }: Props) {
+  return (
+    <article className="space-y-2">
+      <h2 className="text-xl font-bold">{article.title}</h2>
+      <div dangerouslySetInnerHTML={{ __html: article.content ?? '' }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)